### DISCUSSION TITLE
Github student developer pack application keeps getting rejected(for re-applying)

### DISCUSSION THREAD

**Discussion by daniyal-ahmedd** [community/community#63122] [GitHub Education]
### Hi everyone,

I'm trying to submit proof that you have to do once a year and I have tried to submit for verification 2 times. First with my personal email and then with my university email. Both times I've been rejected, even though I as approved a year ago.
![image](https://github.com/community/community/assets/110684391/516cd66a-5524-498f-9e28-01b83d59faf9)

**Comment by humairshoukat** [comment]
You don't have to submit every year until your developer pack has been ended

**Reply by daniyal-ahmedd** [reply]
Its for reverifying academic affiliation. So, that's why I'm doing that.

**Reply by humairshoukat** [reply]
I would say contact a GitHub Campus Expert and ask him about this issue. He/She can raise an internal issue on your behalf to resolve this issue.

**Reply by daniyal-ahmedd** [reply]
How can i contact them? Can you guide me?

**Comment by ianchb** [comment]
> I would say contact a GitHub Campus Expert and ask him about this issue. He/She can raise an internal issue on your behalf to resolve this issue.

Same question - how to do that? I got rejected for more than 80 times this year. :/
